shotover-proxy/src/server.rs

```diff
@@ -253,6 +253,24 @@ impl<C: Codec + 'static> TcpCodecListener<C> {
         }
     }
 
+    pub async fn shutdown(&mut self) {
+        match self
+            .chain
+            .shutdown(Wrapper::new_with_chain_name(
+                vec![],
```

Comment:
oh right we just pass an empty vec into the message_wrapper anyway. Let's remove that from the shutdown call